eng/ci/templates/jobs/run-integration-tests-windows.yml

```diff
@@ -0,0 +1,66 @@
+jobs:
+- job: RunIntegrationTests_Windows
+  displayName: '[Windows] Run Integration Tests'
+
+  pool:
+    name: 1es-pool-azfunc
+    image: 1es-windows-2022
+    os: windows
+
+  steps:
+  - task: NuGetAuthenticate@1
+    displayName: NuGet Authenticate
+
+  - pwsh: . "tools/start-emulators.ps1" -NoWait
+    displayName: "Start emulators (NoWait)"
+
+  - template: /eng/ci/templates/steps/install-dotnet.yml@self
+
+  - task: DotNetCoreCLI@2
+    displayName: Build DotNetWorker
+    inputs:
+      command: build
+      arguments: '-c Release'
+      projects: DotNetWorker.sln
+
+  - pwsh: |
+      $useIntegrationTestingCoreTools = $null
+      if (-not ([bool]::TryParse($env:USECORETOOLSBUILDFROMINTEGRATIONTESTS, [ref] $useIntegrationTestingCoreTools)))
+      {
+        throw "UseCoreToolsBuildFromIntegrationTests can only be set to True or False. Current value is set to $env:USECORETOOLSBUILDFROMINTEGRATIONTESTS"
+      }
+      ./setup-e2e-tests.ps1 -DotnetVersion 'net7' `
+                            -UseCoreToolsBuildFromIntegrationTests:$useIntegrationTestingCoreTools `
+                            -SkipBuildOnPack
+    displayName: '[NET 7] Setup E2E tests'
+
+  - task: DotNetCoreCLI@2
```

Comment:
Consider pulling framework into a template parameter, then call this template twice: once for net7, once for netfx. (And then deleting the duplicate pwsh and test steps)

Reply:
Or maybe strategy/matric will work here also:
https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/jobs-job-strategy?view=azure-pipelines